ilc/common/wrapWithCache.js

```diff
@@ -63,17 +64,16 @@ const wrapWithCache =
                                 cachedAt: now,
                             }),
                         );
-
                         logger.info(
                             {
                                 memoName: name,
                                 hash,
                                 now,
                                 cachedAt: parsedItem ? parsedItem.cachedAt : undefined,
+                                id: contextStore && contextStore.get('reqId'),
```

Comment:
do you have an example of result log?

Reply:
The case is `contextStore` is not available on client side.

Reply:
hmm, `id` format is strange. I thought that we can use it per our internal log specification, but with such format is impossible.
Also please note that having `id` in logs might not be transparent without context of this PR